gwas-fasp/v1.90p/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Base image
+FROM ubuntu:bionic
+
+# Metadata
+LABEL base_image="ubuntu:bionic"
+LABEL version="1"
+LABEL software="PLINK"
```

Comment:
Software label name should match directory name, gaws-fasp in your case

Reply:
Thanks - makes sense!